web/app/models.py

```diff
@@ -226,7 +227,7 @@ def __str__(self):
 class TournamentPlayer(db.Model):
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     name = db.Column(db.String(80))
-    player_id = db.Column(db.String(5))
+    aga_num = db.Column(db.String(5))
     member_ex_date = db.Column(db.String(20))
     rating = db.Column(db.String(5))
```

Comment:
Do you want to simply leave it without a constraint for now or shall we constrain it a 6, 7,...?